### ISSUE TITLE
Feature request: CommonMark attribute syntax

### ISSUE THREAD

**Issue by hektormisplon** [euclio/aurelius#22]
Not sure if this is the place to make this request, but I thought it could be useful to have the  syntax provided by the [CommonMark AttributesExtension](https://commonmark.thephpleague.com/2.3/extensions/attributes/) available.

I am not sure if [pulldown-cmark](https://github.com/raphlinus/pulldown-cmark) already supports this (they do support attributes on headings).

I frequently use this on images and currently [vim-markdown-composer](https://github.com/euclio/vim-markdown-composer) does not handle this kind of syntax.
E.g. when adding a class attribute to an image: `![](img/image_2022-06-05-16-47-34.png){.invert}`. The attribute part is currently rendered as the string `"{.invert}"`.

**Comment by euclio** [comment]
Yep, we're limited by pulldown-cmark here. For vim-markdown-composer you could try using an external renderer that does have support, though: https://github.com/euclio/vim-markdown-composer/blob/5b79f425ebd28216d9aa472be3ba07cda41d9b24/doc/markdown-composer.txt#L31

**Comment by hektormisplon** [comment]
Thank you. I settled with pandoc for now, but it is noticeably slower.
